packages/core/src/interpreter.ts

```diff
@@ -534,6 +534,34 @@ export class Interpreter<
  * @param machine The machine to interpret
```

Comment:
Interpreter is still mentioned in the comments here. It would be cool to also rename this file and the `Interpreter` class itself